app/models/user.rb

```diff
@@ -10,4 +10,5 @@ class User < ApplicationRecord
   validates :first_name, presence: true
   validates :last_name, presence: true
   before_save :skip_confirmation!
+  has_one :rent, dependent: :destroy
```

Comment:
I think this is a has_many

Reply:
Ok, i didn't understand the domain model, i think that each user can has just only one rent at the same time. Ill change it now